app/models/mixins/per_ems_worker_mixin.rb

```diff
@@ -97,7 +97,7 @@ def queue_name_for_ems(ems)
       return "generic" if ems.kind_of?(Host) && ems.acts_as_ems?
```

Comment:
Sorry, didn't see these comments.  @zeari Can you do a followup PR for further refactorings since I clicked the green button too early?